tests/Model/ValueObject/Contact/ContactPointTest.php

```diff
@@ -57,6 +57,19 @@ public function it_should_be_creatable_without_any_parameters(): void
         $this->assertTrue($contactPoint->isEmpty());
     }
 
+    /**
+     * @test
+     */
+    public function it_should_throw_an_exception_when_parameters_are_string_arrays(): void
```

Comment:
Feel free to remove this test again. There is no real added value here because this is typed parameter behaviour.